common/src/test/scala/uk/ac/wellcome/platform/archive/common/storage/services/StorageManifestServiceTest.scala

```diff
@@ -2,46 +2,47 @@ package uk.ac.wellcome.platform.archive.common.storage.services
 
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.{FunSpec, Matchers}
-import uk.ac.wellcome.platform.archive.common.fixtures.{
-  BagLocationFixtures,
-  FileEntry
-}
-import uk.ac.wellcome.platform.archive.common.ingests.models.{
-  InfrequentAccessStorageProvider,
-  StorageLocation
-}
-import uk.ac.wellcome.platform.archive.common.storage.models.ChecksumAlgorithm
+import uk.ac.wellcome.fixtures.TestWith
+import uk.ac.wellcome.platform.archive.common.fixtures.{BagLocationFixtures, FileEntry}
+import uk.ac.wellcome.platform.archive.common.ingests.models.{InfrequentAccessStorageProvider, StorageLocation}
+import uk.ac.wellcome.platform.archive.common.verify.SHA256
+import uk.ac.wellcome.storage.ObjectLocation
 
-import scala.concurrent.ExecutionContext.Implicits.global
 
 class StorageManifestServiceTest
-    extends FunSpec
+  extends FunSpec
     with Matchers
     with ScalaFutures
     with BagLocationFixtures {
 
-  val service = new StorageManifestService()
+  def withStorageManifestService[R](root: ObjectLocation)(testWith: TestWith[StorageManifestService, R]): R = {
```

Comment:
Why pass in the root location if we aren't going to use it?

I prefer the old way of doing it, given the service was stateless (or has that changed?)

Reply:
This is probably a copypasta fossil. Will remove.